compositor_common/src/scene/builtin_transformations.rs

```diff
@@ -17,41 +18,6 @@ pub enum BuiltinTransformationSpec {
     },
 }
 
-impl BuiltinTransformationSpec {
```

Comment:
I would suggest keeping types at the beginning of the file and `impl` at the end